src/content_scripts/modules/x/commandClassRegister/RefreshCommandListCommand.js

```
@@ -30,9 +31,14 @@ export default class RefreshCommandListCommand extends Command {
    */
   async execute(options) {
     console.log('RefreshCommandListCommand.execute');
-    const cache = new CacheManager(toLightningHostname(this.hostname));
-    await COMMAND_CACHE_KEYS.forEach((key) => cache.clear(key));
-    await new Channel(CHANNEL_REFRESH_COMMANDS).publish();
+    publishCommandLoading(true);
+    try {
+      const cache = new CacheManager(toLightningHostname(this.hostname));
+      await COMMAND_CACHE_KEYS.forEach((key) => cache.clear(key));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)